### ISSUE TITLE
Serilog 2.4.0 doesn't work

### ISSUE THREAD

**Issue by igorskys** [serilog-web/classic#40]
Hello,

I updated my projects with latest serilog. Cleaned packages and after that I started getting Serilog 1.5.0.0 dependency not found exception. Is there workaround for latest version of Serilog?

Thanks!

**Comment by nblumhardt** [comment]
Thanks for the report. It sounds like the NuGet package install process may not have created the required assembly binding redirects; do the instructions at: https://weblog.west-wind.com/posts/2014/Nov/29/Updating-Assembly-Redirects-with-NuGet help?

**Comment by igorskys** [comment]
Thanks for the reply. It helped. I was asked to downgrade Serilog to 2.3.0 and for somereason SerilogWeb.Classic returned be to the same issue. The hint doesn't help.

**Comment by nblumhardt** [comment]
Thanks for the follow-up, good to hear it's working.

**Comment by igorskys** [comment]
Actually it is not working in my specific case. We have sharepoint application. This application doesn't use app.config, it uses custom config path. I added assembly redirect to my custom config, but it didn't help.

Here is failure log.

=== Pre-bind state information === LOG: DisplayName = Serilog, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10  (Fully-specified) LOG: Appbase = file:///C:/inetpub/wwwroot/wss/VirtualDirectories/80/ LOG: Initial PrivatePath = C:\inetpub\wwwroot\wss\VirtualDirectories\80\bin Calling assembly : SerilogWeb.Classic, Version=2.1.0.0, Culture=neutral, PublicKeyToken=9462ddd55fbc0e7f. === LOG: This bind starts in default load context. LOG: Using application configuration file: C:\inetpub\wwwroot\wss\VirtualDirectories\80\web.config LOG: Using host configuration file: C:\Windows\Microsoft.NET\Framework64\v4.0.30319\aspnet.config LOG: Using machine configuration file from C:\Windows\Microsoft.NET\Framework64\v4.0.30319\config\machine.config. LOG: Post-policy reference: Serilog, Version=1.5.0.0, Culture=neutral, PublicKeyToken=24c2f752a8e58a10 LOG: Attempting download of new URL file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/Temporary ASP.NET Files/root/16360c98/2517c5fc/Serilog.DLL. LOG: Attempting download of new URL file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/Temporary ASP.NET Files/root/16360c98/2517c5fc/Serilog/Serilog.DLL. LOG: Attempting download of new URL file:///C:/inetpub/wwwroot/wss/VirtualDirectories/80/bin/Serilog.DLL. WRN: Comparing the assembly name resulted in the mismatch: Major Version ERR: Failed to complete setup of assembly (hr = 0x80131040). Probing terminated.
--


Thank you!

**Comment by nblumhardt** [comment]
Thanks for the update.

Serilog 2 has been out for a long time; I'd be happy if we updated this package to depend directly on Serilog 2.5, and bumped its major version.

Is anyone interested in PRing this? Cheers!

**Comment by igorskys** [comment]
It would be nice to have 👍

**Comment by jmkelly** [comment]
see https://github.com/serilog-web/classic/pull/42

**Comment by nblumhardt** [comment]
Closed via #42.

**Comment by tsimbalar** [comment]
I'm not sure, but I believe there are still traces of older versions of Serilog :-/

I see 
```xml
<dependencies>
      <dependency id="Serilog" version="1.5.7" />
</dependencies>
```

in file https://github.com/serilog-web/classic/blob/master/src/SerilogWeb.Classic/SerilogWeb.Classic.nuspec

And the last published NuGet package declares Serilog v1.5.7+ as a dependency : https://www.nuget.org/packages/SerilogWeb.Classic/3.0.19 

![image](https://user-images.githubusercontent.com/160544/31424032-54e289c2-ae58-11e7-8a09-c8c06858c45c.png)

I believe there should be no `<dependencies>` node in the `.nuspec` file, as `nuget pack projectPath` will already generate it for you, looking at the project's `packages.config` file.
